server/services/v1/database/base_runner.go

```diff
@@ -168,7 +169,7 @@ func (runner *BaseQueryRunner) insertOrReplace(ctx context.Context, tx transacti
 	return ts, allKeys, err
 }
 
-func (runner *BaseQueryRunner) mutateAndValidatePayload(coll *schema.DefaultCollection, mutator mutator, doc []byte) ([]byte, error) {
+func (runner *BaseQueryRunner) mutateAndValidatePayload(ctx context.Context, coll *schema.DefaultCollection, mutator mutator, doc []byte) ([]byte, error) {
 	deserializedDoc, err := util.JSONToMap(doc)
```

Comment:
I am just thinking how to avoid this in case schema sign off is true, this is very expensive...(not for this diff though)

Reply:
If we remember that collection doesn't have defaults then we can do it.

Reply:
yes, that info we have - the other thing is stringsToInt64 for JS.